.travis.yml

```diff
@@ -32,6 +32,56 @@ matrix:
         - XCODE_SDK=watchsimulator
         - XCODE_ACTION=build
         - XCODE_DESTINATION="platform=watchOS Simulator,name=Apple Watch - 38mm"
+    - xcode_scheme: ReactiveObjC-macOS
+      xcode_workspace: ReactiveObjC.xcworkspace
+      env:
+        - XCODE_SDK=macosx
+        - XCODE_ACTION="build test"
+        - XCODE_DESTINATION="arch=x86_64"
+        - XCODE_PLAYGROUND_TARGET="x86_64-apple-macosx10.10"
```

Comment:
`XCODE_PLAYGROUND_TARGET` settings and `script/validate-playground.sh` could/should be removed.